node_tree.py

```diff
@@ -523,15 +479,6 @@ def sv_setattr_with_throttle(self, prop_name, prop_data):
         with self.sv_throttle_tree_update():
             setattr(self, prop_name, prop_data)
 
-    def mark_error(self, err):
-        """
-        marks the with system error color
-        will automaticly be cleared and restored to
-        the old color
-        """
-        ng = self.id_data
-        update_error_nodes(ng, self.name, err)
-
```

Comment:
I'm not sure that this is not used. Did you check?

Reply:
Everything what I was removing I surely checked.

Reply:
![2020-08-25_21-36-14](https://user-images.githubusercontent.com/28003269/91208299-0d360a00-e71b-11ea-8372-cccb51a950f8.png)